routes/api/authRouter.js

```diff
@@ -0,0 +1,96 @@
+const express = require("express");
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+const Joi = require("joi");
+const User = require("../../models/user");
+
+const router = express.Router();
+
+const registerSchema = Joi.object({
+  email: Joi.string().email().required(),
+  password: Joi.string().min(6).required(),
+});
+
+const loginSchema = Joi.object({
+  email: Joi.string().email().required(),
+  password: Joi.string().required(),
+});
+
+router.post("/users/register", async (req, res) => {
+  const { error } = registerSchema.validate(req.body);
+  if (error) {
```

Comment:
functions inside routes must be separated in controller files. And just be invoked here